cmd/fleetctl/apply_test.go

```diff
@@ -3781,7 +3781,7 @@ spec:
     macos_settings:
       enable_disk_encryption: true
 `,
-			wantErr: `Couldn't edit enable_disk_encryption. Neither macOS MDM nor Windows is turned on`,
+			wantOutput: `[+] applied fleet config`,
```

Comment:
Let's add a comment that we allow setting disk encryption ON even if MDM is not configured because of Linux.